lab_6_pipeline/pipeline.py

```diff
@@ -3,9 +3,24 @@
 """
 from pathlib import Path
 from typing import List
+from string import punctuation
 
-from core_utils.article.article import SentenceProtocol
+from core_utils.article.article import SentenceProtocol, split_by_sentence
+from core_utils.article.io import from_raw, to_cleaned
 from core_utils.article.ud import OpencorporaTagProtocol, TagConverter
+from core_utils.constants import ASSETS_PATH
+
+
+class EmptyDirectoryError(Exception):
+    """
+    Daaaaamn yo directory's emptie, maaaan
+    """
+
```

Comment:
conllutoken and conllusentence - what sort of relationship do they have? how is it called from OOP perspective?